core/common/src/main/java/alluxio/security/authentication/PlainSaslTransportProvider.java

```diff
@@ -76,19 +76,22 @@ public TTransport getClientTransport(String username, String password,
 
   @Override
   public TTransportFactory getServerTransportFactory() throws SaslException {
+    return getServerTransportFactory(new Runnable() {
+      @Override
+      public void run() {}
+    });
```

Comment:
We may want to introduce an emptyRunnable constructor similar to the one in Guava 19.